.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file in the repository?

Reply:
@rzakhar what about this and in general about .swiftpm ignoring? In general ignoring .swiftpm is not recommended, but if you like we could ignore it partially or completely